### DISCUSSION TITLE
Nemo - Add "Copy file location" entry to context menu

### DISCUSSION THREAD

**Discussion by adrianlopezgalera** [linuxmint/discussions#72] [Ideas]
Hello. I thought it would be convenient to add the possibility of having an entry in the Nemo context menu so that the user can copy the location of a file or folder when selecting them.

This can so far be done indirectly by copying the path shown in the location sector. However, adding an entry to the context menu would make it easier to copy a file's location within a folder when the user is going to run it in a terminal.

Regards.

**Comment by axel358** [comment]
The Copy item on the context menu does this already
